webpack.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     filename: 'moment-range.js',
     library: 'moment-range',
     libraryTarget: 'umd',
-    path: './dist/',
+    path: __dirname + '/dist/',
```

Comment:
when I run "npm run build" with './dist/' I get:
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "./dist/" is not an absolute path!
I can undo the change but I can't build neither test the library